Services/Certificate/classes/Placeholder/Values/class.ilTestPlaceHolderValues.php

```diff
@@ -1,70 +1,32 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilTestPlaceHolderValues implements ilCertificatePlaceholderValues
 {
-    /**
-     * @var ilDefaultPlaceholderValues
-     */
-    private $defaultPlaceHolderValuesObject;
-
-    /**
-     * @var ilCertificateObjectHelper|null
-     */
-    private $objectHelper;
-
-    /**
-     * @var ilCertificateTestObjectHelper
-     */
-    private $testObjectHelper;
-
-    /**
-     * @var ilCertificateUserObjectHelper
-     */
-    private $userObjectHelper;
-
-    /**
-     * @var ilCertificateUtilHelper|null
-     */
-    private $utilHelper;
-
-    /**
-     * @var ilCertificateLPStatusHelper|null
-     */
-    private $lpStatusHelper;
-
+    private ?ilDefaultPlaceholderValues $defaultPlaceHolderValuesObject;
+    private ?ilCertificateObjectHelper $objectHelper;
+    private ?ilCertificateTestObjectHelper $testObjectHelper;
+    private ?ilCertificateUserObjectHelper $userObjectHelper;
+    private ?ilCertificateUtilHelper $utilHelper;
+    private ?ilCertificateLPStatusHelper $lpStatusHelper;
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.